### ISSUE TITLE
Python 2.7 throws UnicodeEncodeError

### ISSUE THREAD

**Issue by chipkellam** [contentful/rich-text-renderer.py#9]
`from __future__ import unicode_literals`

should be added to the top of every file so that Python 2.7 does not throw a hard error:

```
UnicodeEncodeError rich_text_renderer.block_renderers in render
'ascii' codec can't encode character u'\u2019' in position 200: ordinal not in range(128)
```

**Comment by dlitvakb** [comment]
Hey @chipkellam,

I've created a PR to add support, once it gets reviewed, I will make a patch release.

**Comment by dlitvakb** [comment]
This has been released now as `0.2.3`.

Cheers
